### ISSUE TITLE
[Result Summarizer] FOLDER's multi-org input does not work

### ISSUE THREAD

**Issue by shangw-nvidia** [mlcommons/logging#160]
```bash
python3 -m mlperf_logging.result_summarizer ../training_results_v1.0/{*} training 1.0.0 --csv out.csv
```
works but
```bash
(mlperf-logging) shangw@shangw-dt:~/workspace/logging$ python3 -m mlperf_logging.result_summarizer ../training_results_v1.0/{Google,NVIDIA,Graphcore} training 1.0.0 --csv out.csv
usage: mlperf_logging.result_summarizer [-h] [--werror] [--quiet] [-csv CSV] folder {training,hpc} {0.6.0,0.7.0,1.0.0}
mlperf_logging.result_summarizer: error: argument usage: invalid choice: '../training_results_v1.0/NVIDIA' (choose from 'training', 'hpc')
(mlperf-logging) shangw@shangw-dt:~/workspace/logging$ python3 -m mlperf_logging.result_summarizer ../training_results_v1.0/{*} training 1.0.0 --csv out.csv
```
does not work.

**Comment by emizan76** [comment]
I personally never used that, we can fix it or remove the feature.

**Comment by shangw-nvidia** [comment]
Actually this feature does work if we wrap the "company wildcard" inside quotes, such as 
```bash
python3 -m mlperf_logging.result_summarizer "../training_results_v1.0/{Google,NVIDIA}" training 1.0.0
```
as opposed to
```bash
python3 -m mlperf_logging.result_summarizer ../training_results_v1.0/{Google,NVIDIA} training 1.0.0
```
This might be due to some weird bash magics. Regardless, I updated the readme in PR #170 to make it clear.

**Comment by shangw-nvidia** [comment]
Fixed in #170
